shopinvader_auth_jwt/controllers/main.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 20201 ACSONE SA/NV (http://acsone.eu)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.addons.base_rest.controllers import main
+
+
+class InvaderController(main.RestController):
+
+    _root_path = "/shopinvader_jwt/"
+    _collection_name = "shopinvader.backend"
+    _default_auth = "jwt_shopinvader"
+    _default_save_session = False
+    _default_cors = True
```

Comment:
```suggestion
    _default_cors = "*"
```

If _default_cors is the allowed origins ?
Or keep it empty and let each deployment tune it so we don't open it by default.

Reply:
indeed...  IMO, the allowed origins should be provided by a reverse proxy not by odoo...  Nevertheless we must instruct odoo that cors will be used to be sure that the 'OPTIONS' method is allowed on the rest routes...